src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,37 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Ball {
+    private String color;
+    private int number;
+
+    public Ball() {
+        this.color = new ColorSupplier().getRandomColor();
+        this.number = new Random().nextInt(101);
```

Comment:
The maximum possible number should be 100

Reply:
Yes, the max possible number should be 100, that's why I put '101' in the nextInt() method, because to obtain a random integer from 0 to n (where n = 100, in our case) we shoult use nextInt(n+1). Or I've got it wrong?